Source/WebCore/platform/graphics/MediaPlayerPrivate.cpp

```diff
@@ -51,6 +51,7 @@ const PlatformTimeRanges& MediaPlayerPrivateInterface::seekable() const
 {
     if (maxTimeSeekable() == MediaTime::zeroTime())
         return PlatformTimeRanges::emptyRanges();
+    ASSERT(maxTimeSeekable().isValid());
     m_seekable = { minTimeSeekable(), maxTimeSeekable() };
     return m_seekable;
```

Comment:
Why do we need `m_seekable` if we are recreating it each time?

Reply:
Because we return a reference here.